packages/tools/pie-icons-webc/README.md

```diff
@@ -129,9 +129,24 @@ export default function App() {
 }
 ```
 
-#### TODO: Add Vue usage
+#### Vue
 
-If you require icons for a Vue app, you can either use these icons, or use our native Vue component package – [pie-icons-vue](https://www.npmjs.com/package/@justeattakeaway/pie-icons-vue).
+This package requires Node 18+, therefore, if you are using a lower version of Node please use our native Vue component package – [pie-icons-vue](https://www.npmjs.com/package/@justeattakeaway/pie-icons-vue).
+
+To import from the package root:
+
+```tsx
+import { IconAlertTriangleLarge, IconCalendar } from "@justeattakeaway/pie-icons-webc";
+
+export default function App() {
+  return (
+    <div className="App">
+      <icon-calendar />
+      <icon-alert-triangle-large />
```

Comment:
I don't think custom elements are allowed to be self-closing yet
```suggestion
      <icon-calendar></icon-calendar>
      <icon-alert-triangle-large></icon-alert-triangle-large>
```

Reply:
Self-closing actually worked for me in both examples so maybe this is allowed now? 🎉

Reply:
spoke to @xander-marjoram, agreed to leave for now 👍

Reply:
I would suggest indeed self-closing the tags. The reason for this is custom elements are not void elements (they can content children).

I would be interested in seeing how they worked for you @LTurns but I would still suggest following best practice in the docs here